desktop/assets/texts/en-US/main_menu.ftl

```diff
@@ -36,6 +36,7 @@ bookmarks-menu-manage = Manage Bookmarks...
 
 debug-menu = Debug Tools
 debug-menu-open-stage = View Stage Info
+debug-menu-open-root-movie-clip = View Root MovieClip
```

Comment:
I don't know the "protocol" when adding new translatable  text.
I only added the English translation, hopefully that's enough for starters.

Reply:
That's the exact protocol, rest of translations will come in through Crowdin